### PR TITLE
[SMALLFIX] Use static imports for standard test utilities

### PR DESCRIPTION
**Pull request by Moya-Zhang**
Use static imports for standard test utilities in /alluxio/core/common/src/test/java/alluxio/security/authorization/AclActionsTest.java

### CODE REVIEW

File: core/common/src/test/java/alluxio/security/authorization/AclActionsTest.java [hunk 1]
```diff
@@ -11,7 +11,11 @@
 
 package alluxio.security.authorization;
 
-import org.junit.Assert;
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertFalse;
+import static org.junit.Assert.assertTrue;
+//import org.junit.Assert;
```
Comment (newnius) [inline core/common/src/test/java/alluxio/security/authorization/AclActionsTest.java]: This line is no longer necessary.

Comment (Moya-Zhang) [reply core/common/src/test/java/alluxio/security/authorization/AclActionsTest.java]: > This line is no longer needed.

Thanks for your advice, I have removed it.

Comment (Moya-Zhang) [reply core/common/src/test/java/alluxio/security/authorization/AclActionsTest.java]: But it failed again just because I removed the line?😥

**Comment by newnius** [comment]
/usr/src/alluxio/core/common/src/test/java/alluxio/security/authorization/AclActionsTest.java:16: error: Extra newline

File: core/common/src/test/java/alluxio/security/authorization/AclActionsTest.java [hunk 2]
```diff
@@ -11,7 +11,11 @@
 
 package alluxio.security.authorization;
 
-import org.junit.Assert;
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertFalse;
+import static org.junit.Assert.assertTrue;
+
+
```
Comment (newnius) [inline core/common/src/test/java/alluxio/security/authorization/AclActionsTest.java]: /usr/src/alluxio/core/common/src/test/java/alluxio/security/authorization/AclActionsTest.java:16: error: Extra newline

Comment (Moya-Zhang) [reply core/common/src/test/java/alluxio/security/authorization/AclActionsTest.java]: Thanks, it succeeded this time.

**Comment by apc999** [comment]
alluxio-bot, merge this please

**Pull request merged**